### DISCUSSION TITLE
Seeking to the time before music play will prevent the music from playing

### DISCUSSION THREAD

**Discussion by sekatol** [ppy/osu#30470] [Q&A]
### Type

Game behaviour

### Bug description

- Open a replay or play a map with the autoplay mode. This is for seeking around the map.
- When the music starts to play, hold the left key on keyboard to seek to the very beginning of the map.
- The music disappears when it should be play.

### Screenshots or videos

A video briefly shows the bug
https://github.com/user-attachments/assets/f931820f-72b4-44f7-a924-8d38e021d99c

### Version

2024.1009.1-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17598833/compressed-logs.zip)

**Comment by bdach** [comment]
Cannot reproduce on ubuntu with pulseaudio/alsa. Report is from linux, log mentions pipewire, at which point I'm not really super interested in investigating further. Could be distro issue, could be user misconfiguration of pipewire.

You'll have to isolate this one yourself I'm afraid. Downgrading to discussion.
